### PR TITLE
1.5 updatecert

### PR DESCRIPTION
**Pull request by Apollon77**

### CODE REVIEW

File: lib/setup.js [hunk 1]
```diff
@@ -123,6 +123,22 @@ let objects; // instance
 let States;  // constructor
 let states;  // instance
 
+/** Prints the config file to the console */
```
Comment (AlCalzone) [inline lib/setup.js]: This was moved to here:
https://github.com/ioBroker/ioBroker.js-controller/blob/c6854610b905de782cee49a2eabc1cb2ed18ba17/lib/cli/cliProcess.js#L215
Where is it used from? Maybe it should be in cliTools instead?

Comment (AlCalzone) [reply lib/setup.js]: The github search shows no references to this method...

Comment (Apollon77) [reply lib/setup.js]: yes but not in 1.5.x branch!! in 1.5.x this do not exists, or?

Comment (Apollon77) [reply lib/setup.js]: in 2.x branch all is ok I think, but in 1.5.x branch it was makred as error

Comment (AlCalzone) [reply lib/setup.js]: You are right, I forgot this PR is for 1.5.x

Comment (Apollon77) [reply lib/setup.js]: ... I could seacrh all old 1.5.x commits where it was made wrong, but decided just do it it that way for 1.5.x

**Pull request merged**